src/components/charts/glucose/glucose-history.tsx

```
@@ -38,6 +39,12 @@ export const GlucoseHistoryChart = ({ glucoseLogs, chartConfig, className }: Glu
                             }}
                         >
                             <CartesianGrid vertical={false} />
+                            {glucoseTargets.map(target => (
+                                <ReferenceLine y={target.lowThreshold} stroke="var(--chart-3)" strokeDasharray={"3 3"} strokeWidth={2} label={{ value: "Low threshold", position: "top" }} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)